bedrock/products/views.py

```diff
@@ -15,34 +15,53 @@
 from lib.l10n_utils.fluent import ftl
 
 
-def vpn_allowed_countries():
-    countries = settings.VPN_ALLOWED_COUNTRY_CODES
+VPN_FIXED_MONTHLY_PRICE = 'US$4.99'
+VPN_VARIABLE_MONTHLY_PRICE = '9,99‎ €'
+VPN_VARIABLE_6_MONTH_PRICE = '6,99 €'
+VPN_VARIABLE_12_MONTH_PRICE = '4,99 €'
+VPN_VARIABLE_6_MONTH_PRICE_TOTAL = '41,94 €'
+VPN_VARIABLE_12_MONTH_PRICE_TOTAL = '59,88 €'
```

Comment:
Note for reviewer: these variable prices are just placeholder for now and not yet used in the page.